src/components/TicketStatusFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import { Checkbox, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { TicketSearchParam } from '../api/searchApi';
+import { TicketStatus } from '../types/publication_types/ticket.types';
+import { dataTestId } from '../utils/dataTestIds';
+
+const ticketStatusValues: TicketStatus[] = ['New', 'Pending', 'Closed', 'Completed'];
+
+export const TicketStatusFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const selectedStatuses = searchParams.get(TicketSearchParam.Status)?.split(',') || ticketStatusValues;
+
+  const handleChange = (event: SelectChangeEvent<string[]>) => {
+    const newSelectedStatuses = event.target.value as TicketStatus[];
+
+    if (newSelectedStatuses.length > 0) {
+      searchParams.set(TicketSearchParam.Status, newSelectedStatuses.join(','));
+    } else {
+      searchParams.delete(TicketSearchParam.Status);
+    }
+
+    history.push({ search: searchParams.toString() });
+  };
+
+  return (
+    <FormControl variant="outlined" size="small" sx={{ minWidth: '15rem' }}>
+      <InputLabel>{t('tasks.status')}</InputLabel>
+      <Select
+        data-testid={dataTestId.myPage.myMessages.ticketStatusField}
+        multiple
+        value={selectedStatuses}
+        onChange={handleChange}
+        renderValue={(selected) => {
+          if (selected.length === ticketStatusValues.length) {
+            return [t('my_page.messages.all_ticket_types')];
```

Comment:
hvorfor er dette et array? 🤔 
```suggestion
            return t('my_page.messages.all_ticket_types')
```

Reply:
Hadde det for meg at `<Select />` var veldig streng på å ha arrau som render